framework/scenario/src/facade/scenario_world.rs

```diff
@@ -145,7 +145,7 @@ impl ScenarioWorld {
         self.get_mut_debugger_backend()
             .vm_runner
             .contract_map_ref
-            .borrow_mut()
+            .borrow()
```

Comment:
Why is `borrow_mut` no longer necessary? This should be a mutable operation.

Reply:
with Mutex there is only a lock mechanism, no difference between borrow and borrow_mut like there is on RefCell, I'll rename the function to reflect that

Reply:
done by renaming the function "borrow" to "lock" to better match the RefCell -> Mutex conversion

"lock" = both old "borrow" and "borrow_mut" functions